LIPs/LIP-1.md

```diff
@@ -0,0 +1,88 @@
+past revisions.
+
+LIPs can be used to track the progress of a client implementation and its compatibility with the current version of the Livepeer protocol. Implementers can list LIPs that
+have been implemented in a particular client.
+
+## Specification
+
+### LIP Types
+
+There are three types of LIPs:
+
+- A **Standard Track LIP** describes any changes that affect the Livepeer protocol. Currently, these changes are focused around the Ethereum smart contracts and clients
+that interact with the contracts. However, in the future Standard Track LIPs may be further categorized to capture other components of the Livepeer protocol such as
+the networking protocol.
+- An **Informational LIP** provides general guidelines or information to the Livepeer community, but does not propose a new feature. Informational LIPs do not
+necessarily represent Livepeer community consensus or a recommendation, so users and implementers are free to ignore Informational LIPs or follow the outlined advice.
+- A **Meta LIP** describes processes surrounding Livepeer including proposals to change processes. Examples of processes that could be described in Meta LIPs are
+decision-making processes used in the governance around future Livepeer protocol upgrades and tools/environments used in Livepeer development.
+
+### LIP Statuses
+
+- **Draft**: a LIP that is open for consideration.
+- **Accepted**: a LIP that is planned for immediate adoption (i.e. expected to be included in an upcoming protocol upgrade).
+- **Final**: a LIP that has been adopted in a previous protocol upgrade.
+- **Deferred**: a LIP that is not being considered for immediate adoption, but may be reconsidered in the future for a subsequent protocol upgrade.
+- **Rejected**: a LIP that is not being considered for immediate adoption, and will not be reconsidered in the future for a subsequent protocol upgrade.
+
+### LIP Work Flow
+
+The LIP repository editors change the status of LIPs.
+
+The LIP process begins with a new idea for the Livepeer protocol. LIPs should be as focused as possible such that they contain a single key proposal or new idea.
+The LIP editor reserves the right to reject LIP proposals if they appear to be unfocused or too broad.
+
+Each LIP must have a champion - someone who writes the LIP using the appropriate style and format, shepherds discussions in the appropriate forums, and attempts to build
+community consensus around the proposed idea.
+
+Parties interested in submitting a LIP are encouraged to vet an idea publicly before actually writing the proposal to save time and to ensure that the idea presented is
+applicable to the entire Livepeer community. Examples of appropriate public forums to gauge interest around an LIP include [the Livepeer subreddit](https://www.reddit.com/r/livepeer), [the issues for this repository](https://github.com/livepeer/LIPs/issues), [the Livepeer Discord channels](https://discord.gg/7wRSUGX) and [the Livepeer forum](https://forum.livepeer.org/).
```

Comment:
can we switch the ordering to be 1. issues, 2. forum, 3. discord, 4. reddit? I think leading with reddit may be misleading since it's not very active.